src/pages/myAccount/__tests__/MyAccount.test.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { shallow } from "enzyme";
+import MyAccount from "../MyAccount";
+
+// test component rendering
+const currentUser = { userId: "1" };
+const sidebarLinks = [
```

Comment:
It may make sense to import the actual links rather than rewrite them here. you could add an export object to the MyAccount  file called exportedForTestingOnly - eg.
```
export const exportedForTestingOnly = {
    sidebarLinks: sidebarLinks
}
```

This can be included at the very bottom of the file and only used for testing. This way, if the links object changes, the test will still work as expected.

Then in this test file:
```
import ......

const sidebarLinks = exportedForTestingOnly.sidebarLinks
```

Reply:
As the url inside 'sidebarLinks' contain currentUser which is a global state. For example  to: `/my-account/${currentUser.userId}`.
So the 'export const exportedForTestingOnly = {
    sidebarLinks: sidebarLinks
}'  would not work if it was outside the component.
Is there any way to make it work??